lib/karafka/consumers/includer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Karafka
+  # Additional functionalities for consumers
+  module Consumers
+    # Module used to inject functionalities into a given consumer class, based on the consumer
+    # topic and its settings
+    # We don't need all the behaviors in all the cases, so it is totally not worth having
```

Comment:
We don't need all the behaviors in all the cases, so it is not worth having everything in all the cases all the time